Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1051,12 +1051,18 @@ def handle_submitted_notable(service: client.Service, notable: Notable, enrichme
                     job = client.Job(service=service, sid=enrichment.id)
                     if job.is_done():
                         demisto.debug(f'Handling open {enrichment.type} enrichment for notable {notable.id}')
-                        for item in results.JSONResultsReader(job.results(output_mode=OUTPUT_MODE_JSON)):
+                        all_items = results.JSONResultsReader(job.results(output_mode=OUTPUT_MODE_JSON))
+                        demisto.debug(f'{notable.id} {enrichment.type} successful. {len(all_items)}')
+                        for item in all_items:
```

Comment:
`results.JSONResultsReader` return an `generator`
so you will fail in `len(generator)`

Reply:
fixed